src/App.vue

```diff
@@ -1,27 +1,28 @@
 <template>
-  <img alt="Vue logo" src="./assets/logo.png">
+  <!-- <img alt="Vue logo" src="./assets/logo.png"> -->
   <!-- <HelloWorld msg="Welcome to Your Vue.js App" /> -->
-  <!-- <header class="header" v-bind:title="title"></header> -->
-  <Footer class="footer" v-bind:message="footerMessage"></Footer>
+  <Header class="header" v-bind:title="title"></Header>
```

Comment:
can we move `class="header"` inside component right? you can keep class here but if class="header" is always needed when Header component is loaded then better to keep this in component itself right? also we can add a prop class here if any new class is needed which is not consistent at places.